src/functionaltests/java/com/ericsson/gerrit/plugins/eiffel/linking/GerritMock.java

```diff
@@ -0,0 +1,129 @@
+package com.ericsson.gerrit.plugins.eiffel.linking;
+
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import com.ericsson.gerrit.plugins.eiffel.git.CommitInformation;
+import com.ericsson.gerrit.plugins.eiffel.linking.GerritMock.ChangeInfo;
+import com.google.gerrit.extensions.restapi.ResourceNotFoundException;
+
+public class GerritMock {
```

Comment:
Please add Javadoc for all public methods and classes. Please add javadoc for private classes that is not easy to understand.

Reply:
Fixed in 8d44d79